modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,451 @@
+
+[id="known-issues-1-15_{context}"]
+== Known issues
+
+* The new `skopeo-copy` task, which is available from the `openshift-pipelines` namespace by using the cluster resolver, does not work when the `VERBOSE` parameter is set to `false`, which is the default setting. As a workaround, when you use this task, set the `VERBOSE` parameter to `true`. The issue does not apply to the `skopeo-copy` `ClusterTask`.
+
+* The new `skopeo-copy` task, which is available from the `openshift-pipelines` namespace by using the cluster resolver, fails when you use it to push or pull an image to or from an {OCP} internal image repository, such as `image-registry.openshift-image-registry.svc:5000`. As a workaround, set the `DEST_TLS_VERIFY` or `SRC_TLS_VERIFY` parameter to `false`. Alternatively, use an external image repository that has a valid SSL certificate. The issue does not apply to the `skopeo-copy` `ClusterTask`.
+
+* The new `s2i-pass:[*]` tasks, which are available from the `openshift-pipelines` namespace by using the cluster resolver, fail if you clone a Git tepository to a subdirectory of the `source` workspace and then set the `CONTEXT` parameter of the task. As a workaround, when you use these tasks, do not set the `CONTEXT` parameter. The issue does not apply to the `s2i-pass:[*]` `ClusterTasks`.
+
+* The new `git-clone` task, which is available from the `openshift-pipelines` namespace by using the cluster resolver, does not set the `COMMIT` result value. The issue does not apply to the `git-clone` `ClusterTask`.
+
+* The `jib-maven` `ClusterTask` does not work if you are using {OCP} version 4.16.
+
+* When using {pac}, if you set the `concurrency_limit` spec in the global `Repository` CR named `pipelines-as-code` in the `openshift-pipelines` namespace, which provides default settings for all `Repository` CRs, the {pac} watcher crashes. As a workaround, do not set this spec in this CR. Instead, set the `concurrency_limit` spec in the other `Repository` CRs that you create.
+
+* When using {pac}, if you set the `settings.pipelinerun_provenance` spec in the global `Repository` CR named `pipelines-as-code` in the `openshift-pipelines` namespace, which provides default settings for all `Repository` CRs, the {pac} controller crashes. As a workaround, do not set this spec in this CR. Instead, set the `settings.pipelinerun_provenance` spec in the other `Repository` CRs that you create.
+
+[id="fixed-issues-1-15_{context}"]
+== Fixed issues
+
+* Before this update, many info messages about `ClusterTask` resources being repeatedly reconciled were present in the {pipelines-shortname} Operator log. With this update, the excessive reconciliation no longer happens and the excessive messages do not appear.
++
+If the reconciliation messages still appear, remove the old version of the ClusterTask `installerset` resource. However, if you remove the `installerset` resource, you cannot reference `ClusterTasks` with this specified version in your pipelines.
```

Comment:
* "earlier" per ISG
* Still need to mark up `ClusterTask`, but a suggestion for a minor rewrite to avoid two backtick'd words being back-to-back
```suggestion
If the reconciliation messages still appear, remove the earlier version of the `ClusterTask` resource named `installerset`. However, if you remove the `installerset` resource, you cannot reference `ClusterTasks` with this specified version in your pipelines.
```

Reply:
Unfortunately this is not a `ClusterTask` resource named `installerset`. It is an `installerset` resource that installs `ClusterTask` resources . I guess two backticks following each other aren't critical.